x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/list_page/components/create_agent_policy.tsx

```diff
@@ -111,48 +119,68 @@ export const CreateAgentPolicyFlyout: React.FunctionComponent<Props> = ({
           </EuiButtonEmpty>
         </EuiFlexItem>
         <EuiFlexItem grow={false}>
-          <EuiButton
-            fill
-            isLoading={isLoading}
-            isDisabled={!hasFleetAllPrivileges || isLoading || Object.keys(validation).length > 0}
-            onClick={async () => {
-              setIsLoading(true);
-              try {
-                const { data, error } = await createAgentPolicy();
-                setIsLoading(false);
-                if (data) {
-                  notifications.toasts.addSuccess(
-                    i18n.translate('xpack.fleet.createAgentPolicy.successNotificationTitle', {
-                      defaultMessage: "Agent policy '{name}' created",
-                      values: { name: agentPolicy.name },
-                    })
-                  );
-                  onClose(data.item);
-                } else {
-                  notifications.toasts.addDanger(
-                    error
-                      ? error.message
-                      : i18n.translate('xpack.fleet.createAgentPolicy.errorNotificationTitle', {
-                          defaultMessage: 'Unable to create agent policy',
-                        })
-                  );
+          <EuiFlexGroup gutterSize="none">
+            {showDevtoolsRequest ? (
+              <EuiFlexItem grow={false}>
+                <DevtoolsRequestFlyoutButton
+                  isDisabled={isLoading || Object.keys(validation).length > 0}
+                  description={i18n.translate(
+                    'xpack.fleet.createAgentPolicy.devtoolsRequestDescription',
+                    {
+                      defaultMessage: 'This Kibana request will create a new agent policy.',
```

Comment:
nit: the message could say `creates` instead of `will create` to be consistent with the other screens